Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/Introduction.js

```diff
@@ -1,37 +1,44 @@
 import React from 'react';
 import Typography from '@material-ui/core/Typography';
-import { Editor } from '@tinymce/tinymce-react';
+import SunEditor from "suneditor-react";
+import 'suneditor/dist/css/suneditor.min.css';
+import htmlToText from 'html-to-text';
 
 export default function Introduction() {
-  
-  let handleEditorChange = (content, editor) => {
+
+  let handleChange = (content) => {
     //TODO Implement
-    console.log('HTML String:', content);
-    console.log('Plain Text:', editor.getContent({format: 'text'}));
-  }
+    console.log(content);
+    console.log(htmlToText.fromString(content));
+  };
+
   return (
     <div>
       <Typography variant="h4" >
         Introduction:
       </Typography>
-      <Editor
-        apiKey="sd525gpjpg57p2irp3p3nsdjmxj0j708hdfxp3iu0fdw3455"
-        initialValue="<p>Write in your introduction for your component.</p>"
-        init={{
-          height: 350,
-          menubar: true,
-          plugins: [
-            'advlist autolink lists link image charmap print preview anchor',
-            'searchreplace visualblocks code fullscreen',
-            'insertdatetime media table paste code help wordcount'
-          ],
-          toolbar:
-            'undo redo | formatselect | bold italic backcolor | \
-            alignleft aligncenter alignright alignjustify | \
-            bullist numlist outdent indent | removeformat | help'
+      <SunEditor 
+        setOptions={{
+          width:'100%',
+          height: 400,
```

Comment:
You probably want to do something more responsive than just hard coded 400px. Chrome lets you simulate a phone to test stuff like that. Not for this PR, but there are also some small issues with login and the drawer on mobile.

Reply:
Can you share a screenshot of the "login" on mobile?

Reply:
Sorry, I actually meant the landing page 😅 Nothing major, just the top buttons could use some smaller text / better spacing. There is a library I like to use called react-device-detect I think that has a func called isMobile()
<img width="421" alt="Screen Shot 2020-10-07 at 3 23 23 PM" src="https://user-images.githubusercontent.com/47537417/95378046-5dbf8c00-08b1-11eb-9173-ee179ade63d7.png">
<img width="426" alt="Screen Shot 2020-10-07 at 3 24 02 PM" src="https://user-images.githubusercontent.com/47537417/95378047-5dbf8c00-08b1-11eb-921b-5377991cb046.png">

Reply:
A responsive sidebar should be implemented in another PR, I definitely can adjust the buttons in this PR though.

Reply:
Don't worry about either if you don't want for this PR, just wanted to mention it as I saw it!